Assets/Scripts/Multiplayer/CoopActive.cs

```diff
@@ -0,0 +1,13 @@
+using UnityEngine;
+
+namespace Multiplayer
+{
+    public class CoopActive : MonoBehaviour
```

Comment:
Just curious: That class surely makes sense, but what is it used for?

Reply:
you can tag all multiplayer elements with this and it enables/disables that element according to game mode